orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/DeleteScalingPolicyStage.groovy

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.kato.pipeline
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.MonitorKatoTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.ServerGroupCacheForceRefreshTask
+import com.netflix.spinnaker.orca.kato.tasks.DeleteScalingPolicyTask
+import com.netflix.spinnaker.orca.pipeline.LinearStage
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.batch.core.Step
+import org.springframework.stereotype.Component
+
+@Component
+class DeleteScalingPolicyStage extends LinearStage {
+
+  public static final String PIPELINE_CONFIG_TYPE = "deleteScalingPolicy"
+
+  DeleteScalingPolicyStage() {
+    super(PIPELINE_CONFIG_TYPE)
+  }
+
+  @Override
+  public List<Step> buildSteps(Stage stage) {
+    def step1 = buildStep(stage, "deleteScalingPolicy", DeleteScalingPolicyTask)
```

Comment:
Stylistically, I usually just throw them all into the list right from the getgo:

```
[ 
  buildStep(...),
  buildStep(...),
  buildStep(...),
]